<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesLight</class>
 <widget class="QWidget" name="PropertiesLight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>42</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxType">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>70</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Directional</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Point</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spot</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="toolButtonColor">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>100</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>180</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonColorEyeDropper">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>100</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/eyedropper.png</normaloff>:/res/images/eyedropper.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labelColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAngle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>187</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Angle</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxAngle">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>185</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>179</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderAngle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>187</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>179</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelRange">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>158</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxRange">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntensity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Intensity</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxIntensity">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLighting">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>13</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Realtime</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Baked</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelLighting">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lighting</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShadows">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>43</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shadows</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxShadows">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Soft</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="../gameplay-editor.qrc"/>
 </resources>
 <connections/>
</ui>
